### PR TITLE
Tests/float

### PR DESCRIPTION
**Pull request by dbarwikowski**
# Info
This Pull Request is related to issue no. 75 (#75)

### CODE REVIEW

File: tests/Valit.Tests/Float/Float_IsEqualTo_Tests.cs [hunk 1]
```diff
@@ -0,0 +1,146 @@
+﻿using Shouldly;
+using System;
+using Xunit;
+
+namespace Valit.Tests.Float
+{
+    public class Float_IsEqualTo_Tests
+    {
+        [Fact]
+        public void Int16_IsEqualTo_For_Not_Nullable_Values_Throws_When_Null_Rule_Is_Given()
```
Comment (GooRiOn) [inline tests/Valit.Tests/Float/Float_IsEqualTo_Tests.cs]: Int16? DID YOU SIMPLY COPIED THAT? :O

**Review by GooRiOn — CHANGES_REQUESTED** [review]

File: tests/Valit.Tests/Float/Float_IsEqualTo_Tests.cs (hunk 1 above)
Comment (dbarwikowski) [reply tests/Valit.Tests/Float/Float_IsEqualTo_Tests.cs]: Of course not ;)

File: tests/Valit.Tests/Float/Float_IsGreaterThanOrEqualTo_Tests.cs [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        [InlineData(false, 10, true)]
+        [InlineData(false, 11, false)]
+        [InlineData(true, 10, false)]
+        [InlineData(false, Single.NaN, false)]
+        [InlineData(true, Single.NaN, false)]
+        public void Float_IsGreaterThanOrEqualTo_Returns_Proper_Results_For_Nullable_Value_And_Not_Nullable_Value(bool useNullValue, float value, bool expected)
+        {
+            IValitResult result = ValitRules<Model>
+                .Create()
+                .Ensure(m => useNullValue ? m.NullValue : m.NullableValue, _ => _
```
Comment (GooRiOn) [inline tests/Valit.Tests/Float/Float_IsGreaterThanOrEqualTo_Tests.cs:160]: You don't check Nullable NaN as left side opperand

File: tests/Valit.Tests/Float/Float_IsGreaterThanOrEqualTo_Tests.cs [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+        [Theory]
+        [InlineData(9, true)]
+        [InlineData(10, true)]
+        [InlineData(11, false)]
+        [InlineData(Single.NaN, false)]
+        public void Float_IsGreaterThanOrEqualTo_Returns_Proper_Results_For_Not_Nullable_Values(float value, bool expected)
+        {
+            IValitResult result = ValitRules<Model>
+                .Create()
+                .Ensure(m => m.Value, _ => _
```
Comment (GooRiOn) [inline tests/Valit.Tests/Float/Float_IsGreaterThanOrEqualTo_Tests.cs:123]: You don't check NaN as left side operand

**Review by GooRiOn — CHANGES_REQUESTED** [review]

**Pull request merged**